walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +125,37 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+   @Test
+  public void everyAttendeeIsConsideredOptional() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times.
+    //           |--------------C--------------|
+    // Events  :       |--A--|     |--B--|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
```

Comment:
Consider creating a setUp() method that handles all logic common to all tests -- in this case, adding events to a collection (pass in correct events as parameters).  More readable and modular.